kahuna/public/js/mixpanel/mixpanel.js

```diff
@@ -47,6 +46,6 @@ mp.factory('mixpanel', ['$window', 'mixpanelEnabled', function($window, mixpanel
     return {
         init: init,
         track: track,
-        isEnabled: () => mixpanelEnabled
+        isEnabled: () => initialised
```

Comment:
Fair point. After some experimenting I figured it's actually better to keep that state in the analytics service. Have a look at the latest commits and let me know if you think that's better!